src/rest-server/src/models/v2/job/k8s.js

```diff
@@ -868,7 +903,10 @@ const put = async (frameworkName, config, rawConfig) => {
   const auths = Object.values(config.prerequisites.dockerimage)
     .filter((dockerimage) => dockerimage.auth != null)
     .map((dockerimage) => dockerimage.auth);
-  auths.length && await createSecret(frameworkName, auths);
+  auths.length && await createDockerSecret(frameworkName, auths);
```

Comment:
I think we need a more complete exception catch logic. We should store each k8s objects we created. And try to delete all created objects if error happened

Reply:
As offline discuss, will leave GC to watch dog